strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NpmArtifactGenerator.java

```diff
@@ -112,14 +110,7 @@ private void calculateChecksum()
         throws IOException
     {
         MessageDigest crypt;
-        try
-        {
-            crypt = MessageDigest.getInstance("SHA-1");
-        }
-        catch (NoSuchAlgorithmException e)
-        {
-            throw new UndeclaredThrowableException(e);
```

Comment:
we can replace it with `throw new IOException(e)` instead of using `ThrowingFunction`